homeassistant/components/lg_thinq/strings.json

```diff
@@ -267,16 +287,31 @@
       "growth_mode_for_location": {
         "name": "{location} mode",
         "state": {
-          "standard": "[%key:component::lg_thinq::entity::sensor::growth_mode::state::standard%]",
-          "ext_leaf": "[%key:component::lg_thinq::entity::sensor::growth_mode::state::ext_leaf%]",
-          "ext_herb": "[%key:component::lg_thinq::entity::sensor::growth_mode::state::ext_herb%]",
-          "ext_flower": "[%key:component::lg_thinq::entity::sensor::growth_mode::state::ext_flower%]",
-          "ext_expert": "[%key:component::lg_thinq::entity::sensor::growth_mode::state::ext_expert%]"
+          "standard": "Auto",
+          "ext_leaf": "Vegetables",
+          "ext_herb": "Herbs",
+          "ext_flower": "Flowers",
+          "ext_expert": "Custom growing mode"
         }
       },
+      "wind_volume": {
+        "name": "Wind speed"
+      },
```

Comment:
If the source entity gets the wind speed device class this can be omitted

Reply:
Since same key _for_location exist, I want to maintain the "wind_volume".

Reply:
Why exactly? You can just remove this one and save translations for our contributors